data/quests.ts

```diff
@@ -933,12 +954,12 @@ export const Quests: Array<ListType> = [
       },
       {
         id: "f0f2c1df-5dd3-458e-b919-7d1f7af26196",
-        description: "Get Finger Slayer Blade in Nokron.",
+        description: "Get Fingerslayer Blade in Nokron.",
       },
       {
         id: "05e68320-e745-4472-8fd0-7364c001249c",
         description:
-          "Give the item to Ranni in Ranni's Rise. You will receive the inverted statue",
+          "Give the item to Ranni in Ranni's Rise. You will receive the inverted statue (Note: You will not be able to complete Seluvis's quest after this point).",
```

Comment:
I don't really like the note like I said in another comment but I have added some things like this in code comments to maybe one day provide some sort of information and have it display on another line or something. I don't know exactly what to do with it. I think for now this is fine just shorten it like this.

```suggestion
          "Give the item to Ranni in Ranni's Rise. You will receive the inverted statue (You will not be able to complete Seluvis's quest after this point).",
```